docs: hepa-instruction-manual-rollers-update

jwwojak opened the pull request:
# Overview

This PR does 2 things:

1. It updates the HEPA module instruction manual to include the new rollers (images and text).
2. It removes H1 headers to help reduce search clutter. The `nav` list in the main YAML file will control the page titles.

Opening as draft until we can get change date for the rollers.

See RTC-763.

## Test Plan and Hands on Testing

Not much to test, does it work?

## Changelog

In the `product-specifications.md` file, added an H2 section at the end that describes the roller feature. We're waiting on a manufacture date (when did this change happen) for this feature. It does look odd as an H2 in here. Not sure where else to put it. Alternative: just use the replacement labeled part image and leave it at that, no new text. The new feature is easy to understand and may not require explanation.

Removed all H1 headers from all the pages. Will let the `mkdocs.yml` file control the page titles.

Updated labeled module image. We're inconsistent in the docs w/ render vs illustration for this particular (non-cover page) image. Went w/ render.

## Review requests

Pls. take a look and comment.

## Risk assessment

Trivial to low.

jwwojak committed:
Door rollers and removing H1 headers

Adds new images showing the door rollers. Removed H1 headers to reduce search results clutter.

jwwojak committed:
Door rollers and removing H1 headers

Adds new images showing the door rollers. Removed H1 headers to reduce search results clutter.

jwwojak committed:
Door rollers and removing H1 headers

Adds new images showing the door rollers. Removed H1 headers to reduce search results clutter.

jwwojak committed:
Punctuation

jwwojak committed:
Punctuation

jwwojak committed:
Punctuation

jwwojak committed:
Update text and image for module rollers

- What happened in the past: module scratches door.

- What's the fix: rollers

- When: July 2027, no need to mention serial number to tell if a module has rollers or not. It's too obvious.

jwwojak committed:
Update text and image for module rollers

- What happened in the past: module scratches door.

- What's the fix: rollers

- When: July 2027, no need to mention serial number to tell if a module has rollers or not. It's too obvious.

ecormany commented on docs/hepa-uv/docs/images/hepa-uv-hero.png:
Hmm, is this the biggest version of the new image we have? It's small enough that it's being enlarged past its pixel dimensions at the default size when rendered on the site (on a high DPI monitor like a MacBook internal screen). And the shadow is cut off at the right edge.

jwwojak replied:
The drop shadow is my cropping it too closely. Also, I don't know what this means:

> It's small enough that it's being enlarged past its pixel dimensions at the default size when rendered on the site

What should I do to correct that? Let me try with the illustrations. There are no drop shadows and they're larger.

jwwojak committed:
Replace renders with illustrations

For the door rollers, removing illustrations and problem drop-shadows for illustrations.

We could also try with just captions instead of the text w/ pointers. These are full size, original images. Letting browser manage the display size.

jwwojak committed:
Replace renders with illustrations

For the door rollers, removing illustrations and problem drop-shadows for illustrations.

We could also try with just captions instead of the text w/ pointers. These are full size, original images. Letting browser manage the display size.

jwwojak committed:
Revising images

- New cover hero
- New labeled parts image
- New illustration for parts list and door rollers

There are some older model images in here, in the assembly steps and an image that shows air filtration.

I do not feel comfortable asking Jackson for updates to these images. They still work and convey the idea as intended even without showing rollers.

Let's hold our image requests for the vacuum manifold.

jwwojak committed:
Revising images

- New cover hero
- New labeled parts image
- New illustration for parts list and door rollers

There are some older model images in here, in the assembly steps and an image that shows air filtration.

I do not feel comfortable asking Jackson for updates to these images. They still work and convey the idea as intended even without showing rollers.

Let's hold our image requests for the vacuum manifold.

jwwojak committed:
Merge branch 'hepa-rollers-update' of https://github.com/Opentrons/opentrons into hepa-rollers-update

jwwojak commented:
Cannot resolve conflicts here via web editor. May rebase branch and try.

jwwojak committed:
Merge branch 'hepa-rollers-update' of https://github.com/Opentrons/opentrons into hepa-rollers-update

jwwojak committed:
Disaster recovery

jwwojak commented:
Closing and re-opening under new management.
